core/server/proxy/pom.xml

```diff
@@ -74,6 +74,10 @@
       <groupId>org.glassfish.jersey.inject</groupId>
       <artifactId>jersey-hk2</artifactId>
     </dependency>
+    <dependency>
```

Comment:
@rongrong @HelloHorizon did either of you check the license for this new dependency?

Reply:
Refer to this page https://mvnrepository.com/artifact/org.glassfish.jersey.inject/jersey-hk2, there isn't a red flag for this dependency